### DISCUSSION TITLE
如何上传example的截图

### DISCUSSION THREAD

**Discussion by zhnny** [antvis/L7#2158] [Q&A] [answered]
添加一个example后，如何上传截图，然后在其相关的meta.json里面添加screenshot的地址
例如，添加一个Three.js的示例，如何在[meta.json](https://github.com/antvis/L7/blob/master/packages/site/examples/engine/three/demo/meta.json)里添加screenshot的地址

**Comment by lzxue** [comment, marked as answer]
接触目前是上传到 CDN 的，如果外部用户需要自己找个稳定三方图床。

推荐：或者直接在 PR 的时候回复图片，我统一上传提供 CDN地址
